tools/virtiostat.py

```diff
@@ -194,6 +222,29 @@
 }
 """
 
+# filter for driver name
+if args.driver:
+    bpf_text = bpf_text.replace('DRIVERFILTER',
+        """char filter_driver[] = \"%s\";
+        char driver[16];
+        bpf_probe_read_kernel(driver, sizeof(driver), vq->vdev->dev.driver->name);
```

Comment:
Since vq->vdev->dev.driver->name is really a string, let us use bpf_probe_read_kernel_str here.